src/applications/rated-disabilities/components/AppContent.jsx

```diff
@@ -37,7 +37,7 @@ export default function AppContent() {
   const hasRatedDisabilities = individualRatings?.length > 0;
 
   let contentOrError;
-  if (hasError) {
+  if (hasError || (data && data.errors)) {
```

Comment:
This shouldn't be an issue, but if for some reason the API call doesn't throw in the `useEffect` above, this should add an extra layer of protection